q2_health_index/plugin_setup.py

```diff
@@ -18,12 +19,18 @@
     short_description='Plugin for calculating the Gut Microbiome Health Index (GMHI).'
 )
 
-plugin.visualizers.register_function(
-    function=print_hello,
-    inputs={},
-    parameters={},
-    input_descriptions={},
-    parameter_descriptions={},
-    name='Print hello.',
-    description='Print hello on the screen.'
+plugin.methods.register_function(
+    function=calculate_gmhi,
+    inputs={'table': FeatureTable[RelativeFrequency],
+            'list': '.txt'},
```

Comment:
I think this should be `TSVTaxonomyFormat`

Reply:
Okay, I'll implement it and adapt plugin code in the next commit :)

Reply:
@tkosciol it worked as 

'species = Artifact.import_data('FeatureData[Taxonomy]', q2_h_species, view_type='HeaderlessTSVTaxonomyFormat')

def species_list_from_q2_artifact(feature_data_taxonomy: FeatureData[Taxonomy]) -> List[str]:
    df = feature_data_taxonomy.view(pd.DataFrame)
    species_list = [x for x in df['Taxon']]
    
    return species_list

mh_list = species_list_from_q2_artifact(species)'